acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java

```diff
@@ -111,4 +121,73 @@ public static void checkResourcePerms(List<String> resources) {
     public static boolean needAdminPerm(Integer code) {
         return ADMIN_CODE.contains(code);
     }
+
+    public static void parseResourcePermsAndNamespacePerms(PlainAccessResource plainAccessResource, PlainAccessConfig plainAccessConfig) {
+        List<ResourceAndPerm> resourcePerms = plainAccessConfig.getResourcePerms();
+        if (resourcePerms != null && !resourcePerms.isEmpty()) {
+            for (ResourceAndPerm resource : resourcePerms) {
+                ResourceType type = resource.getType();
+                String namespace = resource.getNamespace();
+                String perm = resource.getPerm();
+                String resourceName = namespace == null ? resource.getResource() : namespace + NAMESPACE_SEPARATOR + resource.getResource();
+                if (type == ResourceType.GROUP) {
+                    resourceName = PlainAccessResource.getRetryTopic(resourceName);
+                }
+                plainAccessResource.addResourceAndPerm(resourceName, Permission.parsePermFromString(perm));
+            }
+        }
+        List<NamespaceAndPerm> namespacePerms = plainAccessConfig.getNamespacePerms();
+        if (namespacePerms != null && !namespacePerms.isEmpty()) {
+            Map<String, Map<String, Byte>> namespacePermMap = new HashMap<>();
+            for (NamespaceAndPerm namespace : namespacePerms) {
+                String namespaceName = namespace.getNamespace();
+                String topicPerm = namespace.getTopicPerm();
+                String groupPerm = namespace.getGroupPerm();
+                Map<String, Byte> permMap = Maps.newHashMapWithExpectedSize(2);
+                if (topicPerm != null && !topicPerm.isEmpty()) {
+                    permMap.put(AclConstants.CONFIG_TOPIC_PERM, Permission.parsePermFromString(topicPerm));
+                }
+                if (groupPerm != null && !groupPerm.isEmpty()) {
+                    permMap.put(AclConstants.CONFIG_GROUP_PERM, Permission.parsePermFromString(groupPerm));
+                }
+                namespacePermMap.put(namespaceName, permMap);
+            }
+            plainAccessResource.setNamespacePermMap(namespacePermMap);
+        }
+    }
+
+    public static void parseResourcePermsAndNamespacePerms(PlainAccessResource plainAccessResource, Map<String, Object> accountMap) {
```

Comment:
Is it almost the same as `parseResourcePermsAndNamespacePerms(org.apache.rocketmq.acl.plain.PlainAccessResource, org.apache.rocketmq.common.PlainAccessConfig)`? If so, just build `PlainAccessConfig` from `Map` and call it.